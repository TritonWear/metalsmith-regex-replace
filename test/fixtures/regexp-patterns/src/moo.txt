I saw one cow talking to another cow. The first cow said "moo", then the
second cow said "moooo". Then they both said "mooooooooooooooo"!