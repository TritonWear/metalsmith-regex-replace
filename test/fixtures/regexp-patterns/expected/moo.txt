I saw one cat talking to another cat. The first cat said "meow", then the
second cat said "meow". Then they both said "meow"!